tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,9 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert len(vendor.inventory) == 3
+    assert result == False
```

Comment:
👍 
I like the length test to ensure that all the items are still there. We don't really need the check for `item` not being in the list, since it was never in the list. And I like the explicit `False` check here, rather than something a little more falsy.

We could still consider checking that each of the three things that _were_ in the list at the start are still there (that there were no shenanigans from the remove), but given that you're checking the overall length already, that's largely a matter of personal preference.